.cursor/mcp.sample.json

```
@@ -0,0 +1,8 @@
+{
+    "mcpServers": {
+      "Figma MCP": {
```

[imo]
MCPを指定するときにこのほうが入力するのが楽になる気がします

```suggestion
      "figma": {
```